Microsoft.Toolkit.Uwp.UI/Media/CanvasBrushBase.cs

```diff
@@ -44,6 +45,11 @@ public abstract class CanvasBrushBase : XamlCompositionBrushBase
         protected override void OnConnected()
         {
             base.OnConnected();
+            CanvasDevice device = CanvasDevice.GetSharedDevice();
+            CompositionGraphicsDevice graphics = CanvasComposition.CreateCompositionGraphicsDevice(Window.Current.Compositor, device);
+
+            device.DeviceLost -= CanvasDevice_DeviceLost;
+            device.DeviceLost += CanvasDevice_DeviceLost;
```

Comment:
Need to also register the `RenderingDeviceReplaced` event

Reply:
I forgot that. It is ready to be reviewed.